README.md

```diff
@@ -40,21 +54,18 @@ More information can be found here: [Ewon Historical Logging](https://developer.
 
 ## Getting Started
 
-### Configuration
-
-User configuration of the application parameters is done through the ConnectorConfig.json file.  This file must be modified to match your configuration.
+### Basic authentication
 
-#### Web Configuration
+This connector uses Basic authentication to communicate with OSIsoft and requires Basic authentication is enabled.
 
-The ConnectorConfig.json file can be edited through the configuration webpage.  The configuration webpage can be accessed after the [installation process](#Installation) has been completed.  The url for the configuration interface **Your_Flexy's_IP_Address**/usr/config.html
+### Configuration
 
-![osisoft web configuration interface](docs/images/configurationPage.png)
+User configuration of the application parameters is done through the ConnectorConfig.json file.  This file must be modified to match your configuration.
 
 #### FTP Configuration
-If you wish to manually edit your ConnectorConfig.json file or you have a saved ConnectorConfig.json file from another Flexy you can load your ConnectorConfig.json file on the flexy in the /usr/ directory via FTP.
+To change configuration values, manually edit your ConnectorConfig.json file on the Flexy in the /usr/ directory via FTP.
```

Comment:
```suggestion
To change configuration values, manually edit the ConnectorConfig.json file on the Flexy in the /usr/ directory via FTP.
```

Are you editing via FTP or updating the file via FTP and editing with another tool? FTP is not an editor.